src/main/resources/hudson/plugins/jira/Messages.properties

```diff
@@ -33,3 +33,4 @@ CommentStep.Descriptor.DisplayName=JIRA: Add a comment to issue(s)
 SearchIssuesStep.Descriptor.DisplayName=JIRA: Search issues
 IssueSelectorStep.Descriptor.DisplayName=JIRA: Issue selector
 JiraCreateIssueNotifier.DisplayName=JIRA: Create issue
+FindIssuesAtCommitLogsStep.Descriptor.DisplayName=JIRA: find all issues referenced by the commit logs of this build
```

Comment:
I would remove `JIRA:` as it is going to be shown as `jiraFindIssuesAtCommitLogs: JIRA: find all issues referenced by the commit logs of this build`. The repeated `:` looks weird (although all other steps in this plugin seems to be using the same display message pattern...)